dev/inside_out/notebook.clj

```diff
@@ -68,7 +68,7 @@
 ;; Let's make this interactive with a text `:input`.
 
 (cljs
- (with-form [contact-info {:person/name ?name}]
```

Comment:
without a namespace to these references things don't seem to resolve in `cljs` blocks ![20221129_16h29m20s_grim](https://user-images.githubusercontent.com/94817/204575182-460165ee-ca68-459b-bc39-fba410ffb1ea.png)

not sure if this is actually the best way to deal with the issue though